frontend/pages/donations/make.tsx

```diff
@@ -45,7 +91,9 @@ const Donate: FC<DonateProps> = (props: DonateProps) => {
         });
         if (response.data == "OK") {
           router.push("/donations/success");
+          // return <SuccessMessage />;
```

Comment:
@sunzae99, if we use state (suggested in https://github.com/ModularHistory/modularhistory/pull/382/files#r629693380), we could do this:
```
if (response.data == "OK") {
    setSuccess(true);
} else {
    setError(true);
}
```